src/mame/mame.lst

```diff
@@ -16078,6 +16078,9 @@ sk10                            //
 @source:casio/sx1000.cpp
 sx1010                          // Casio SX1010
 
+@source:casio/zoomer.cpp
+zoomer                          // 1993 Casio/Tandy
```

Comment:
This thing was also sold with GRiD and AST branding (as the GridPad 2390).  Is the firmware dump you’ve used Tandy-branded, or is it generic Casio firmware?  I wouldn’t try stuffing all the marketing names into the description and manufacturer if they all used the same firmware.

Reply:
I dumped it from a Casio unit, but the same mask ROMs appear in the parts list for the Tandy version. I don't know about the GridPad.

Reply:
Does the Casio XL7000 Zoomer variant have the same ROMs, too?  (IIRC the Z-7000 dropped the “Zoomer” branding, just keeping the stylised “Z” logo on the outside of the flip cover.)

Reply:
Also, while Casio didn't use the Zoomer name in packaging/marketing, the device itself still refers to itself as such all over the place, e.g.: 
![0000](https://github.com/user-attachments/assets/91e622e7-753d-41f9-b7a6-ab709d4de6db)